### PR TITLE
BPTC (BC6H and BC7) compressed texture support

### PR DESCRIPTION
**Pull request by ddenisyuk**
##### Description of change
Added support of BPTC(BC6H and BC7) compressed textures. 
See: 
- https://www.khronos.org/opengl/wiki/BPTC_Texture_Compression
- https://developer.mozilla.org/en-US/docs/Web/API/EXT_texture_compression_bptc
- https://registry.khronos.org/OpenGL/extensions/EXT/EXT_texture_compression_bptc.txt

##### Pre-Merge Checklist

- [x] Tests and/or benchmarks are included
- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by ddenisyuk** [comment]
@bigtimebuddy Is there something more needed to be merged?

File: packages/basis/src/loader/BasisParser.ts [hunk 1]
```diff
@@ -316,6 +316,7 @@ export class BasisParser
                 pvrtc: gl.getExtension('WEBGL_compressed_texture_pvrtc')
                     || gl.getExtension('WEBKIT_WEBGL_compressed_texture_pvrtc'),
                 atc: gl.getExtension('WEBGL_compressed_texture_atc'),
+                bptc: gl.getExtension('EXT_texture_compression_bptc'),
```
Comment (querielo) [inline packages/basis/src/loader/BasisParser.ts:319]: Given that it provides one of the highest texture qualities, I believe it's better to declare 
`bptc: gl.getExtension('EXT_texture_compression_bptc'),` as high as possible.

**Comment by bigtimebuddy** [comment]
@ddenisyuk I'm trying to merge this but getting blocked by a test. Any ideas?

**Review by bigtimebuddy — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
Thank you @ddenisyuk :tada:

File: packages/basis/src/loader/BasisParser.ts (hunk 1 above)
Comment (ddenisyuk) [reply packages/basis/src/loader/BasisParser.ts:319]: I've created a separate issue for that. 
https://github.com/pixijs/pixijs/issues/10213

**Pull request merged**